content/about/faq.md

```diff
@@ -2,48 +2,86 @@
 {{</ faq >}}
 
 {{< faq "Will the presentations be public after the conference?" >}}
-Yes. Our main goal is to foster FOSS4G, and that means also making all the presentations available even to people that didn’t register in the first place. After the conference, the recorded talks will be published on our YouTube channel - https://www.youtube.com/@FOSS4G  
+Yes, all presentations will be recorded and made publicly available after the
+conference. Our main goal is to foster FOSS4G, and that means also making all
+the presentations available even to people that didn’t register in the first
+place. After the conference, the recorded talks will be published on our
+YouTube channel - https://www.youtube.com/@FOSS4G  
 {{</ faq >}}
 
 {{< faq "Have you considered a hybrid online-face to face event?" >}}
-FOSS4GE 2025 will be an in-person event.  
+Yes, FOSS4G Europe 2025 is organized as an in-person event. However, to
+accommodate a wider audience, we plan to provide online streaming of the
+conference sessions. This approach allows geospatial enthusiasts who cannot
+attend in person to follow the talks and participate virtually.  
 {{</ faq >}}
 
 {{< faq "I am not dedicated to geoinformatics, can I attend the conference?" >}}
-Yes, the conference is open to any public interested in geoinformatics, free software, maps, and communities.  
+Absolutely. The conference welcomes anyone interested in geospatial technologies
+and open-source software. Whether you're a professional from another field, a
+student, or simply curious about geospatial innovations, your participation is
+encouraged.  
 {{</ faq >}}
 
 {{< faq "If I don’t develop or write code, is this conference for me, too?" >}}
-Yes, this event is for all kinds of people interested in sharing experiences about maps, from developers who write code to users of geoinformatics software, people who make decisions, researchers, students, and anyone who wants to learn, share and meet like-minded people.  
+Yes, the conference is designed for a diverse audience, including users,
+educators, and decision-makers. You don't need to be a developer or coder to
+benefit from the sessions and networking opportunities. The event offers
+insights into the application and impact of geospatial technologies across
+various sectors.  
 {{</ faq >}}
 
 {{< faq "What activities will the conference have?" >}}
-The conference will have tracks full of talks, workshops, and community sprint activities. Social programme is very diverse from Geochicas event to Ice-Breaker and Social Bytes.
+The conference will feature a variety of activities, including:
+
+- **Workshops:** Hands-on sessions led by experts to enhance your skills in
+open-source geospatial technologies.
+- **Presentations:** Talks and lectures covering the latest advancements and
+applications in the geospatial field.
+- **Networking Events:** Opportunities to connect with professionals from
+around the world, such as GeoBiz Connect and GeoGals Night.
```

Comment:
"GeoGals"... I'll try this again as we have already talked about it a couple of times. 

If this means "GeoChicas take Mostar", then please don't rename the event. Unless the GeoChicasOSM leads tell us that they would like to have a different name for the event. Remember it's a networking event by women for women (people identifying as women+allies). Our task (as men) is to support it by providing a safe space, be there if needed and make it possible for [GeoChicasOSM](https://wiki.openstreetmap.org/wiki/LatAm/Groups/GeoChicas) to organize the thing.

Secondly, I think calling grownup people "girls" in a professional setting is shauvinist and belittleing (unless they themselves choose to do it). And this sort of attitude should not be supported anywhere, most of all in an open community setting that a FOSS4G is.